Sources/Maker.swift

```diff
@@ -24,6 +24,15 @@ public var nui_safeArea: SafeArea {
     return SafeArea()
 }
 
+public enum Sides {
```

Comment:
I suppose we need to add docs to this enum. For example when we should use it and how. Also we need explanation for each case in it